CMakeLists.txt

```diff
@@ -95,6 +95,9 @@ mark_as_advanced(UA_ENABLE_STATUSCODE_DESCRIPTIONS)
 option(UA_ENABLE_TYPENAMES "Add the type and member names to the UA_DataType structure" ON)
 mark_as_advanced(UA_ENABLE_TYPENAMES)
 
+option(UA_ENABLE_NAMESPACE_DESCRIPTIONS "Add the description of namespace entry" ON)
```

Comment:
One small change request here:
It's probably better to name it `UA_ENABLE_NODESET_COMPILER_DESCRIPTIONS`

Can you clarify the option a bit more:

"Set node description attribute for nodeset compiler generated nodes"

Reply:
No problem, this is now fixed. Should I merge the two commits to one? not sure how that impacts the pull request(?).